src/MakePotCommand.php

```diff
@@ -293,28 +291,39 @@ protected function get_main_file_data() {
 	/**
 	 * Creates a POT file and stores it on disk.
 	 *
-	 * @return bool True on success, false otherwise.
+	 * @return Translations A Translation set.
 	 */
-	protected function makepot() {
-		$this->translations = new Translations();
+	protected function extract_strings() {
+		$translations = new Translations();
 
 		// Add existing strings first but don't keep headers.
 		if ( $this->merge ) {
 			$existing_translations = new Translations();
 			Po::fromFile( $this->merge, $existing_translations );
-			$this->translations->mergeWith( $existing_translations, Merge::ADD | Merge::REMOVE );
+			$translations->mergeWith( $existing_translations, Merge::ADD | Merge::REMOVE );
 		}
 
 		$meta = $this->get_meta_data();
 		PotGenerator::setCommentBeforeHeaders( $meta['comments'] );
 
-		$this->set_default_headers();
+		$meta = $this->get_meta_data();
+
+		$translations->setHeader( 'Project-Id-Version', $meta['name'] . ' ' . $meta['version'] );
```

Comment:
Was there a specific reason why this block was inlined instead of remaining in a separate method? I'd consider this is a regression in terms of code quality.

Reply:
I suggested it in an inline comment. I wanted to some refactoring in another PR after this one.

Reply:
Because it was requested^^
I restored the function.

Reply:
Ah, sorry @swissspidy, I missed that comment.